.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Feature Request
+about: Please first, look at existing issues to see if the feature has been requested before.
+labels: enhancement
+---
+Please first, look at [existing issues](https://github.com/openzipkin/zipkin-ruby/issues) to see if the feature has been requested before. If you don't find anything tell us what problem you’re trying to solve. Often a solution already exists! Don’t send pull requests to implement new features without first getting our support. Sometimes we leave features out on purpose to keep the project small.
+
+**Feature:**
+Description of the feature
+
+**Rational**
+Why would this feature help others besides me?
+
+**Example Scenario**
+What use case is not practical to serve before this feature?
```

Comment:
Reads a bit clunky IMHO, how about something like "What kind of use cases would benefit from this feature?"

Reply:
agree it is awkward, I was trying to rephrase brain into considering if it is actually possible now or not, but instead, just broke the brain!